src/Altinn.Apps/AppFrontend/react/package.json

```diff
@@ -1,27 +1,27 @@
 {
-  "name": "app-frontend",
-  "private": true,
-  "scripts": {
-    "install-deps": "lerna bootstrap --hoist --ci",
-    "no-ci-hoist": "lerna bootstrap --hoist --no-ci",
-    "install-deps-with-lock": "lerna bootstrap",
-    "test": "lerna exec --concurrency 1 -- npm test",
-    "install-all": "lerna exec -- npm ci",
-    "clean-lock": "lerna exec -- rm ./package-lock.json",
-    "clean-node-modules": "rm -rf ./node_modules && lerna exec --concurrency 1 -- rm -rf ./node_modules",
-    "update-packages": "lerna exec --concurrency 1 -- npm update --dev",
-    "list-outdated": "lerna run list-outdated"
-  },
-  "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "~2.26.0",
-    "@typescript-eslint/parser": "~2.26.0",
-    "eslint": "~6.8.0",
-    "eslint-config-airbnb": "~18.1.0",
-    "eslint-plugin-import": "~2.20.2",
-    "eslint-plugin-jsx-a11y": "~6.2.3",
-    "eslint-plugin-react": "~7.19.0",
-    "eslint-plugin-react-hooks": "~2.5.0",
-    "ini": "1.3.5",
-    "lerna": "3.20.2"
-  }
+	"name": "app-frontend",
+	"private": true,
+	"scripts": {
+		"install-deps": "lerna bootstrap --hoist --ci",
```

Comment:
Why so much indent?

Reply:
No idea. Looks correct in VS, I'm suspecting this is a presentation-error.